### ISSUE TITLE
Better Readme with API examples

### ISSUE THREAD

**Issue by FZambia** [centrifugal/jscent#3]
Looks like I missed this before, time to fill the gap.

**Comment by synw** [comment]
Yes more examples regarding to client token generation in various languages would be good for the new users I guess: I remember the first times I read the documentation the token generation part looked a bit obscure to me, hopefully the well designed python library was to the rescue.
